components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -91,13 +88,31 @@ class GleanTest {
         assertEquals(path, "/submit/$applicationId/test/${Glean.SCHEMA_VERSION}/$uuid")
     }
 
+    @Test
+    fun `test experiments recording`() {
+        Glean.setExperimentActive(
+            "experiment_test", "branch_a"
+        )
+        Glean.setExperimentActive(
+            "experiment_api", "branch_b",
+            mapOf("test_key" to "value")
+        )
+        Glean.setExperimentInactive("experiment_test")
```

Comment:
Thought: how about another set of assertions before this call, like the ones below?